src/wms/Interfaces/DynamicsCE.ts

```diff
@@ -0,0 +1,12 @@
+export interface FacillitySchedules {
```

Comment:
Do we want to be sticking with pascal case like what is used for EventEntry when defining interface keys

Reply:
Yes, good point. It would be good to be consistent. We'll have to have a chat to decide on what our styling should be, or if DVSA already have a guide.